q2_sample_classifier/citations.bib

```diff
@@ -2,8 +2,11 @@ @article {Bokulich306167
 	author = {Bokulich, Nicholas and Dillon, Matthew and Bolyen, Evan and Kaehler, Benjamin D and Huttley, Gavin A and Caporaso, J Gregory},
 	title = {q2-sample-classifier: machine-learning tools for microbiome classification and regression},
```

Comment:
```suggestion
	title = {{q2-sample-classifier}: machine-learning tools for microbiome classification and regression},
```

This should fix https://github.com/qiime2/docs/issues/341, but would you mind double-checking @nbokulich ?